Simultaneous writes to Environment on GitHub Actions

In the ServiceControl repo there are many separate projects that all must be versioned the same. They rely on `Particular.Packaging` to do the job. 

Occasionally we see this error (or one like it):

```
C:\Users\runneradmin\.nuget\packages\particular.packaging\2.0.0\build\Particular.Packaging.targets(78,5): error MSB3491: Could not write lines to file “D:\a\_temp\_runner_file_commands\set_env_b1b9e0dc-065d-4d67-a4ae-3a1ea65d898c”. The process cannot access the file ‘D:\a\_temp\_runner_file_commands\set_env_b1b9e0dc-065d-4d67-a4ae-3a1ea65d898c’ because it is being used by another process. [D:\a\ServiceControl\ServiceControl\src\ServiceControl.Transports\ServiceControl.Transports.csproj]
```

This is likely caused by [this line](https://github.com/Particular/Particular.Packaging/blob/master/src/Particular.Packaging/build/Particular.Packaging.targets#L78) where we write MinVer version info to the environment variables (which in GitHub Actions is represented as a file). If two projects attempt to execute that action at the same time, there will be a conflict.

--- a/src/Particular.Packaging/build/Particular.Packaging.targets
+++ b/src/Particular.Packaging/build/Particular.Packaging.targets
@@ -66,7 +66,7 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="WriteMinVerProperties" AfterTargets="MinVer" Condition="'$(GITHUB_ACTIONS)' == 'true'">
+  <Target Name="WriteMinVerProperties" AfterTargets="MinVer" Condition="'$(GITHUB_ACTIONS)' == 'true' And '$(WriteMinVerProperties)' != 'false'">
     <ItemGroup>
       <MinVerProperties Include="MinVerVersion=$(MinVerVersion)" />
       <MinVerProperties Include="MinVerMajor=$(MinVerMajor)" />

